nilearn/plotting/displays.py

```diff
@@ -1255,3 +1260,47 @@ def get_slicer(display_mode):
 def get_projector(display_mode):
     "Internal function to retrieve a projector"
     return get_create_display_fun(display_mode, PROJECTORS)
+
+
+def check_threshold(threshold, data, percentile_calculate, name):
+    """
+    Checks that the given threshold has an accepted string value
+    and returns a threshold computed on the data. Use case of data
+    is mainly from plot_connectome and plot_prob_atlas.
+
+    Parameters
+    ----------
+    threshold: a real value or a percentage in string or list of these.
```

Comment:
What I think is that since you've added some tests, your PR is now ready to merge and I don't want to spend weeks arguing about one function. Factorizing this code with the one in `plot_connectome` is not trivial and maybe we should keep that for the sprint.